src/App.js

```diff
@@ -1,26 +1,199 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React in MateAcademy!
-        </a>
-      </header>
-    </div>
+import './fonts/stylesheet.css'
+import './css/App.css';
+
+import TodoItem from './Components/TodoItem';
+import Footer from './Components/Footer';
+
+class App extends React.Component{
+  constructor(){
+    super()
+    this.state = {
+      inputValue: '',
+      tasks: [],
+      activeTasks: [],
+      completedTasks: [],
```

Comment:
Fields `activeTasks` and `completedTasks` are redundant.
Use `Array.prototype.filter()` to filter completed or active tasks